Packaging/nix/devilutionx.6

```diff
@@ -0,0 +1,134 @@
+.TH DEVILUTIONX 6 "May 2025" "1.5.4" "DevilutionX Community"
+.SH NAME
+devilutionx \- A source port of Diablo and Hellfire
+.SH SYNOPSIS
+devilutionx [OPTION]
+.SH DESCRIPTION
+.I DevilutionX
+is a source port of
```

Comment:
We prefer to refer to it as simply a port because of ambiguities around the term.

```suggestion
devilutionx \- A port of Diablo and Hellfire
.SH SYNOPSIS
devilutionx [OPTION]
.SH DESCRIPTION
.I DevilutionX
is a port of
```